vg/vg_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vg_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/plotter"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+	"github.com/gonum/plot/vg/vgeps"
+	"github.com/gonum/plot/vg/vgimg"
+	"github.com/gonum/plot/vg/vgpdf"
+	"github.com/gonum/plot/vg/vgsvg"
+)
+
+// TestLineWidth tests output against test images generated by
+// gen_width.go - these images have been visually inspected and
+// validated.
+func TestLineWidth(t *testing.T) {
+	formats := []string{
+		// TODO: Add logic to cope with run to run eps differences.
```

Comment:
eps includes creation time data.

The logic would involve stripping that from the testdata file and the buffered bytes.